libs/features/mas/web-components/src/variants/twp.css.js

```diff
@@ -0,0 +1,102 @@
+import { TABLET_UP, DESKTOP_UP, LARGE_DESKTOP, MOBILE_LANDSCAPE,} from '../media.js';
+export const CSS = `
+:root {
+  --consonant-merch-card-twp-width: 268px;
+  --consonant-merch-card-twp-mobile-width: 300px;
+  --consonant-merch-card-twp-mobile-height: 358px;
+}
+merch-card[variant="twp"] div[class$='twp-badge'] {
+  padding: 4px 10px 5px 10px;
+}
+
+merch-card[variant="twp"] [slot="body-xs-top"] {
+  font-size: var(--consonant-merch-card-body-xs-font-size);
+  line-height: var(--consonant-merch-card-body-xs-line-height);
+  color: var(--merch-color-grey-80);
+}
+
+merch-card[variant="twp"] [slot="body-xs"] ul {
+  padding: 0;
+  margin: 0;
+}
+
+merch-card[variant="twp"] [slot="body-xs"] ul li {
+  list-style-type: none;
+  padding-left: 0;
+}
+
+merch-card[variant="twp"] [slot="body-xs"] ul li::before {
+  content: '·';
+  font-size: 20px;
+  padding-right: 5px;
+  font-weight: bold;
+}
+
+merch-card[variant="twp"] [slot="footer"] {
+  font-size: var(--consonant-merch-card-body-xs-font-size);
+  line-height: var(--consonant-merch-card-body-xs-line-height);
+  padding: var(--consonant-merch-spacing-s)
+  var(--consonant-merch-spacing-xs) var(--consonant-merch-spacing-xs);
+  color: var(--merch-color-grey-80);
+  display: flex;
+  flex-flow: wrap;
+}
+
+merch-card[variant='twp'] merch-quantity-select,
+merch-card[variant='twp'] merch-offer-select {
+  display: none;
+}
+
+.one-merch-card.twp,
+.two-merch-cards.twp,
+.three-merch-cards.twp {
+  grid-template-columns: var(--consonant-merch-card-image-width);
+}
+
+@media screen and ${DESKTOP_UP} {
```

Comment:
I am looking at the order and I think declarations should follow mobile to large desktop or vice versa, however here we have desktop, large desktop, mobile, tablet, which might get confusing at some point when different CSS is added